### PR TITLE
Update bootstrap inbox

### PR DESCRIPTION
**Pull request by ashwiniHerle**
This PR includes refactoring of the following components:

-  all mydb/inbox components to latest react-bootstrap.
- refactoring CommentList and CommentButton to latest react bootstrap.
 (Other comments-components are either already done/ nothing to be changed)

- mydb/elements/labels/ElementCollectionLabels.js

- mydb/elements/labels/ElementResearchPlanLabels.js

### CODE REVIEW

File: app/packs/src/apps/mydb/inbox/UnsortedDataset.js [hunk 1]
```diff
@@ -145,10 +135,9 @@ export default class UnsortedDataset extends React.Component {
   dropzone() {
     return (
       <Dropzone
-        onDrop={files => this.handleFileDrop(files)}
-        style={{ height: 50, width: '100%', border: '3px dashed lightgray' }}
-      >
-        <div style={{ textAlign: 'center', paddingTop: 12, color: 'gray' }}>
+        onDrop={(files) => this.handleFileDrop(files)}
+        className="border-dashed border-gray-200">
```
Comment (beque) [inline app/packs/src/apps/mydb/inbox/UnsortedDataset.js:139]: The drop zone does not have a 3px border anymore
Is it correct?
Haven't you uses `border-3` at other drop zones?

**Review by beque — COMMENTED** [review]
Only one little question ;-)

File: app/packs/src/apps/mydb/inbox/DatasetContainer.js [hunk 2]
```diff
@@ -73,104 +73,97 @@ class DatasetContainer extends Component {
...
+          {
+            inboxSize && inboxSize !== 'Small'
+            && (
+              <span className={`text-info ms-auto ${largerInbox ? '' : 'none'}`}>
+                {formatDate(dataset.created_at)}
+              </span>
+            )
+          }
+        </ButtonToolbar>
+        <div>{visible ? attachments : null}</div>
```
Comment (maiwald) [inline app/packs/src/apps/mydb/inbox/DatasetContainer.js:163]: Can this be converted to `{visible ?? <div>…}` so we don't render the empty `<div>`?

File: app/packs/src/apps/mydb/inbox/InboxModal.js [hunk 3]
```diff
@@ -429,8 +429,10 @@ export default class InboxModal extends React.Component {
               </Card.Header>
               <Card.Body>
                 <div>
-                  <div className="tree-view">
-                    <div className={`${inboxVisible ? 'd-none' : ''}`}>
+                  <div>
+                    <div
+                      className={`${inboxVisible ? 'd-none' : ''}`}
```
Comment (maiwald) [inline app/packs/src/apps/mydb/inbox/InboxModal.js:434]: Can this be converted to `{inboxVisible && <div…}`?

File: app/packs/src/apps/mydb/inbox/InboxModal.js [hunk 4]
```diff
@@ -441,7 +443,9 @@ export default class InboxModal extends React.Component {
                     </div>
 
                   </div>
-                  <div className={`tree-wrapper${inboxVisible ? 'd-none' : ''}`}>
+                  <div
+                    className={`tree-wrapper${inboxVisible ? 'd-none' : ''}`}
```
Comment (maiwald) [inline app/packs/src/apps/mydb/inbox/InboxModal.js:447]: If this cannot be converted to `{inboxVisible && <div…}`, there is a space missing between the class names. 😊

File: app/packs/src/apps/mydb/inbox/UnsortedDatasetModal.js [hunk 5]
```diff
@@ -7,38 +8,34 @@ import Container from 'src/models/Container';
...
-                  Save
-                </Button>
-              </ButtonToolbar>
-            </Modal.Footer>
           </Modal>
-      ));}
+        </div>
+      )
+    } else {
+      return null;
```
Comment (maiwald) [inline app/packs/src/apps/mydb/inbox/UnsortedDatasetModal.js:36]: NIT: I prefer an early return when `!show` but that is a personal preference.

**Review by maiwald — COMMENTED** [review]
Just minor things 😊

File: app/packs/src/apps/mydb/inbox/UnsortedDataset.js (hunk 1 above)
Comment (ashwiniHerle) [reply app/packs/src/apps/mydb/inbox/UnsortedDataset.js:139]: Yes, I will make it border-3 to be consistent :)

File: app/packs/src/apps/mydb/inbox/UnsortedDatasetModal.js (hunk 5 above)
Comment (ashwiniHerle) [reply app/packs/src/apps/mydb/inbox/UnsortedDatasetModal.js:36]: Yes, looks a lot cleaner, made the changes

**Pull request merged**